docs/icp/17-box-shadow-hover/17-box-shadow-hover.md

```diff
@@ -133,7 +135,276 @@ box-shadow: 5px 5px 5px 2px #808080;
+        background-size: auto;
+      }
+    </style>
+  </head>
+
+  <body>
+    <div class="container">
+      <h1>This is a card</h1>
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Optio
+        voluptates cum veritatis odio voluptas nihil commodi sunt facere illum
+        quis nostrum nam quasi aut ipsam aliquam, similique accusamus nulla!
+        Molestias.
+      </p>
+    </div>
+  </body>
+</html>
+```
+
+**Output :**
+
+<img src="/icp/17/step-5.png" alt="step-4" width="600px"/>
+
+**Example explanation:**
+
+In the above example, we set background image by using `background-img` property. Also we can set `background-size` that sets the original size of the image and image is small that's why it display image repetedly.
+
+### `background-size: cover`
+
+This property is used to cover the area of the container.
+
+**Example :**
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>background-image</title>
+    <style>
+      .container {
+        border: 1px solid black;
+        width: 600px;
+        height: 400px;
+        background-image: url("https://cdn-icons-png.flaticon.com/128/3595/3595455.png");
+        background-size: cover;
+      }
+    </style>
+  </head>
+
+  <body>
+    <div class="container">
+      <h1>This is a card</h1>
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Optio
+        voluptates cum veritatis odio voluptas nihil commodi sunt facere illum
+        quis nostrum nam quasi aut ipsam aliquam, similique accusamus nulla!
+        Molestias.
+      </p>
+    </div>
+  </body>
+</html>
+```
+
+**Output :**
+
+<img src="/icp/17/step-7.png" alt="step-7" width="600px"/>
+
+### We can set background size in height and width:
+
+**Example :**
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>background-image</title>
+    <style>
+      .container {
+        border: 1px solid black;
+        width: 600px;
+        height: 400px;
+        background-image: url("https://cdn-icons-png.flaticon.com/128/3195/3195966.png");
+        background-size: 600px 400px;
+      }
+    </style>
+  </head>
+
+  <body>
+    <div class="container">
+      <h1>This is a card</h1>
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Optio
+        voluptates cum veritatis odio voluptas nihil commodi sunt facere illum
+        quis nostrum nam quasi aut ipsam aliquam, similique accusamus nulla!
+        Molestias.
+      </p>
+    </div>
+  </body>
+</html>
+```
+
+**Output :**
+
+<img src="/icp/17/step-8.png" alt="step-8" width="600px"/>
+
+### Background repeat
+
+This property is used to repeat the image. It can repeat image in differnet direction
+
+`background-repeat: repeat-x;`
```

Comment:
Explain This property 

This value is used to repeat the background image horizontally along the `x-axis`.